rasa/utils/common.py

```diff
@@ -106,8 +110,29 @@ def read_global_config(path: Text) -> Dict[Text, Any]:
         return {}
 
 
+def configure_logging_from_file(logging_config_file: Text) -> None:
+    """Parses YAML file content to configure logging.
+
+    Args:
+        logging_config_file: YAML file containing logging configuration to handle
+            custom formatting
+    """
+    logging_config_dict = rasa.shared.utils.io.read_yaml_file(logging_config_file)
+
+    try:
+        logging.config.dictConfig(logging_config_dict)
+    except (ValueError, TypeError, AttributeError, ImportError):
+        logging.debug(
+            f"The logging config file {logging_config_file} could not "
+            f"be applied because it failed validation against "
+            f"the built-in Python logging schema. "
+            f"More info at {PYTHON_LOGGING_SCHEMA_DOCS}."
+        )
```

Comment:
this would swallow the original error, right? this would leave the user without the information to debug their config. Should we log the error?